www/pages/_document.tsx

```diff
@@ -87,7 +98,11 @@ const clientSideJS = `
   document.addEventListener('DOMContentLoaded', event => {
     document.querySelector('select[name=language]').addEventListener('change', () => document.tune.submit())
     document.querySelector('select[name=time]').addEventListener('change', () => document.tune.submit())
+  })
+`;
 
+const serviceWorkerRegistration = `
+  document.addEventListener('DOMContentLoaded', event => {
     if ('serviceWorker' in navigator) {
       window.addEventListener('load', () => {
         navigator.serviceWorker.register('/_next/static/service-worker.js', { scope: "/" }).then(registration => {
```

Comment:
i'm not registering service worker in dev mode to avoid console error about missing file